sdks/go/pkg/beam/core/runtime/harness/session.go

```diff
@@ -33,15 +33,12 @@ import (
 var capture io.WriteCloser
 
 var (
```

Comment:
nit for a separate PR: Since we removed the flag to enable the session runner, we might as well remove all the associated code too.